images/toolbox/toolbox.sh

```diff
@@ -48,23 +79,15 @@ else
 export ROOK_API_SERVER_ENDPOINT=${ROOK_API_SERVER_ENDPOINT}
 
 EOF
-
-    CLIENT_INFO=$(curl -s http://${ROOK_API_SERVER_ENDPOINT}/client)
-    MON_ENDPOINTS=$(echo $CLIENT_INFO | jq -r '.monAddresses[]' | awk -F '/' 'BEGIN { ORS="," }; {print $1}' | sed 's/,$//')
-    KEYRING_FILE="/etc/ceph/keyring"
-
-    cat <<EOF > ${CEPH_CONFIG}
-[global]
-mon_host = ${MON_ENDPOINTS}
-
-[client.admin]
-keyring = ${KEYRING_FILE}
-EOF
-
+    
     cat <<EOF > ${KEYRING_FILE}
 [client.admin]
 key = ${ROOK_ADMIN_SECRET}
 EOF
+
+    write_endpoints
+    watch_endpoints
```

Comment:
this watch_endpoints function will never return since it has an endless loop, right?  that means the `exec ${ARGS}` call will never happen.  That's not necessarily an issue right now since the `rook-tools.yaml` just passes a long sleep, but it could be an issue if someone is using a different arg in their yaml.

Reply:
what if we don't support the args? this means i can remove the `sleep` args from the yaml as well. You're just expected to `exec` to the pod if you want to execute something.